dialogue-core/src/test/java/com/palantir/dialogue/core/ContentDecodingChannelTest.java

```diff
@@ -57,12 +58,15 @@ void before() throws Exception {
                 ImmutableList.of("https://localhost:8123"),
                 SSLContext.getDefault().getSocketFactory(),
                 tm);
+        List<TargetUri> targets = ImmutableList.of(TargetUri.of("https://localhost:8123"));
         standard = Mockito.mock(Config.class);
         Mockito.when(standard.mesh()).thenReturn(MeshMode.DEFAULT_NO_MESH);
         Mockito.when(standard.clientConf()).thenReturn(clientConfig);
+        Mockito.when(standard.uris()).thenReturn(targets);
         mesh = Mockito.mock(Config.class);
         Mockito.when(mesh.mesh()).thenReturn(MeshMode.USE_EXTERNAL_MESH);
         Mockito.when(mesh.clientConf()).thenReturn(clientConfig);
+        Mockito.when(mesh.uris()).thenReturn(targets);
```

Comment:
Using a mocked config instead of building the `Config` instances directly meant we had to make code changes. The defaults in the Config class would have taken care of this automatically using the real type.